src/transformers/models/llama/modeling_llama.py

```diff
@@ -225,7 +225,12 @@ def forward(
                 )
             attn_weights = attn_weights + attention_mask
             attn_weights = torch.max(
-                attn_weights, torch.tensor(torch.finfo(attn_weights.dtype).min, device=attn_weights.device)
+                attn_weights,
+                torch.tensor(
+                    torch.finfo(attn_weights.dtype).min, 
+                    device=attn_weights.device, 
+                    dtype=attn_weights.dtype
+                ),
```

Comment:
```suggestion
dtype_max = torch.tensor(torch.finfo(attn_weights.dtype).min,device=attn_weights.device,dtype=attn_weights.dtype)
attn_weights = torch.max(attn_weights,dtype_max)
```
It should hold in two line. If it does not then it's fine!

Reply:
THis change + `make style` should be good 😉